.github/workflows/nightly.yml

```diff
@@ -0,0 +1,35 @@
+name: Next Publication
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - lm-nightly-release
```

Comment:
nit: Do we want this branch here? Is workflow dispatch not good enough?

Reply:
I'm testing after the changes. I'll remove it when all works